### PR TITLE
New VEP plugin IntAct (e107)

### PR DESCRIPTION
**Pull request by nakib103**
### Description

The VEP endpoint will have the external plugin IntAct available. This documentation is needed for the plugin.

### Use case

Plugin functionality available through REST.

### Benefits

The endpoint needs documentation to be displayed.

### Possible Drawbacks

None, the default behavior is the same.

### Testing

note to submitters and reviewers: documentation-only changes may reflect
changes in other repos that can result in new or different output from
REST endpoints. In turn, these may require new tests or changes to existing tests.

_Have you added/modified unit tests to test the changes?_

_If so, do the tests pass/fail?_

_Have you run the entire test suite and no regression was detected?_

### Changelog

_Are you changing the functionality of an endpoint? If so, please give a one line summary for the public facing changelog._

eg. [/xenobiology/orthologs] Added the ability to look up orthologs and paralogs from klingons and andorians
[/vep/:species/hgvs/] new plugin option added: IntAct
[/vep/:species/id/] new plugin option added: IntAct
[/vep/:species/region/] new plugin option added: IntAct

### CODE REVIEW

**Comment by nakib103** [comment]
@dglemos can you please have a look?

File: root/documentation/vep.conf [hunk 1]
```diff
@@ -154,6 +154,10 @@
         description=Detects splice sites in genomic DNA (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/GeneSplicer.pm">plugin details</a>)
         default=0
       </GeneSplicer>
+      <IntAct>
+        type=Boolean
+        description=Provides molecular interaction data for variants as reprted by <a target="_blank" href="https://www.ebi.ac.uk/intact/home">IntAct</a> database. (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/master/IntAct.pm">plugin details</a>)
```
Comment (dglemos) [inline root/documentation/vep.conf:159]: There's a typo 'reprted'

Comment (nakib103) [reply root/documentation/vep.conf]: Thanks! I have given fix.

**Comment by TamaraNaboulsi** [comment]
Hi @nakib103 , could you please add the changelog lines related to this change (if any)? As we need that for branching this repo. You can check the example given in the PR template, or check #517 for a similar change. Thanks!

**Review by dglemos — APPROVED** [review]

**Comment by nakib103** [comment]
> Hi @nakib103 , could you please add the changelog lines related to this change (if any)? As we need that for branching this repo. You can check the example given in the PR template, or check #517 for a similar change. Thanks!

Hello @TamaraNaboulsi , sorry missed that - I have updated the Changelog

**Review by TamaraNaboulsi — APPROVED** [review]
Adding IntAct plugin to VEP id, region, and hgvs (get and post). Looks good.

**Pull request merged**